### ISSUE TITLE
Some animation transitions may make content inaccessible to ChromeVox users

### ISSUE THREAD

**Issue by aitchiss** [forem/forem#12939]
**Describe the bug**

We recently became aware that ChromeVox (the in-built screen reading software for Chromebook) does not announce content that is set to have `opacity: 0` (even if the opacity is then later transitioned to opacity > 0 via e.g. an animation) (See https://twitter.com/sarasoueidan/status/1365696682529333253)

We have a few instances in the app where we use `opacity: 0` as the starting point for a transition - e.g. the `sidebar-wrapper` in `articles.scss`.

**To Reproduce**

I haven't actually been able to reproduce this as it seems ChromeVox is only available as the native screen reader on Chromebooks.

**Expected behavior**

Elements that fade or animate in should still be accessible to ChromeVox users. We can address this by adding a very small opacity to the elements that are currently 0, e.g. `opacity: 0.00001`

**Comment by cmgorton** [comment]
I believe #12939 closes this issue.
